client/src/components/SharedComponents/navbar/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
```

Comment:
You don't need Component here as you can make the component a function. Have a look at how the others have done it for the right syntax, should have no render and no explicit return.